library/src/blas3/rocblas_trtri_strided_batched.cpp

```diff
@@ -0,0 +1,193 @@
+        if(!n || !batch_count)
+            return rocblas_status_success;
+
+        rocblas_status status;
+        if(n <= NB)
+        {
+            status = rocblas_trtri_small<NB, T>(handle,
+                                                uplo,
+                                                diag,
+                                                n,
+                                                A,
+                                                0,
+                                                lda,
+                                                bsa,
+                                                0,
+                                                invA,
+                                                0,
+                                                ldinvA,
+                                                bsinvA,
+                                                0,
+                                                batch_count,
+                                                1);
+        }
+        else
+        {
+            // Compute the optimal size for temporary device memory
+            size_t size = rocblas_trtri_temp_size<NB>(n, batch_count) * sizeof(T);
+
+            // If size is queried, set optimal size
+            if(handle->is_device_memory_size_query())
+                return handle->set_optimal_device_memory_size(size);
+
+            // Allocate memory
+            auto C_tmp = handle->device_malloc(size);
+            if(!C_tmp)
+                return rocblas_status_memory_error;
+
+            status = rocblas_trtri_large<NB, false, true, T>(handle,
+                                                             uplo,
+                                                             diag,
+                                                             n,
+                                                             A,
+                                                             0,
+                                                             lda,
+                                                             bsa,
+                                                             0,
+                                                             invA,
+                                                             0,
+                                                             ldinvA,
+                                                             bsinvA,
+                                                             0,
+                                                             batch_count,
+                                                             1,
+                                                             (T*)C_tmp);
+        }
+
+        return status;
+    }
+
+}
+
+/* ============================================================================================ */
+
+/*
+ * ===========================================================================
+ *    C interface
+ *    This function is called by trsm
```

Comment:
` *    This function is called by trsm`

Typo?

Reply:
Looks like this was copied from the original trtri_batched.cpp file, will remove.